ts/Dashboard/SerializeHelper/HTMLTableStoreHelper.ts

```diff
@@ -46,8 +48,10 @@ const { merge } = U;
 function fromJSON(
     json: HTMLTableStoreHelper.JSON
 ): HTMLTableStore {
-    const table = DataTableHelper.fromJSON(json.table),
-        store = new HTMLTableStore(table, json.options);
+    const converter = // HTMLTableConverterHelper.fromJSON(json.converter),
```

Comment:
Remove the comment. Do we need this `any`?

Reply:
This casting is temporary, it will be replaced with `HTMLTableConverterHelper` when it will get implemented.